fe/src/main/java/org/apache/doris/external/EsStateStore.java

```diff
@@ -17,6 +17,16 @@
 
 package org.apache.doris.external;
 
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.Maps;
+import com.google.common.collect.Range;
+import okhttp3.Authenticator;
+import okhttp3.Call;
+import okhttp3.Credentials;
```

Comment:
import order should be:

org.apache.doris
com
org
java
javax

change the setting of your java IDE